tests/infra/libbyz/node.py

```diff
@@ -55,7 +55,11 @@ def stop(self):
         logger.info(f"IP: {self.machine_name} - terminating process")
         if self.proc:
             self.proc.terminate()
-            self.proc.wait()
+            result = self.proc.wait(5)
+            if result != 0:
+                self.proc.kill()
+                self.proc.wait()
+
```

Comment:
I was thinking maybe we could `cat` the `out\<port\>.txt` files when we know something isn't right. We can do that here and if the `syscall_stats` assertion is going to fail.

Reply:
this is a bigger work item #734